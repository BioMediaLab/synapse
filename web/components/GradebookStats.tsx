import React from "react";

export default () => <div>Gradebook Stats</div>;
