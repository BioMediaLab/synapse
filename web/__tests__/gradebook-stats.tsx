import React from "react";
import { render } from "react-testing-library";
import GradebookStats from "../components/GradebookStats";

test("mounts", () => {
  const { container } = render(<GradebookStats />);
  expect(container).toMatchSnapshot();
});
