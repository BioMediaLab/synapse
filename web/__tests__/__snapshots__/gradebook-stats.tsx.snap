// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mounts 1`] = `
<div>
  <div>
    Gradebook Stats
  </div>
</div>
`;
